src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml

```diff
@@ -167,46 +167,88 @@
                 <h2>
                     <ui:msg description="Professional Services - Trusted By">Trusted By</ui:msg>
                 </h2>
-                <slider:Slider>
-                    <slider:content>
-                        <g:HTMLPanel>
-                            <ul>
-                                <li>
+                <div class="{res.style.mobileHidden}">
+                    <slider:Slider>
+                        <slider:content>
+                            <g:HTMLPanel>
+                                <ul>
+                                    <li>
+                                        <a href="http://zafin.com/" target="_blank">
+                                            <g:Image resource="{page.protorisk}"/>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#">
+                                            <g:Image resource="{page.kaam}"/>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="http://www.lillegroup.com/" target="_blank">
+                                            <g:Image resource="{page.swiftIQ}"/>
+                                        </a>
+                                    </li>
+                                </ul>
+                                <ul>
+                                    <li>
+                                        <a href="#">
+                                            <g:Image resource="{page.easyData}"/>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="http://zafin.com/" target="_blank">
+                                            <g:Image resource="{page.bookedIn}"/>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="http://www.lillegroup.com/" target="_blank">
+                                            <g:Image resource="{page.trackR}"/>
+                                        </a>
+                                    </li>
+                                </ul>
+                            </g:HTMLPanel>
+                        </slider:content>
+                    </slider:Slider>
+                </div>
+
+                <div class="{res.style.mobileVisible}">
```

Comment:
When I apply the addStyleNames on the slider, the build is failing, I'll ask sp to check that out.